evalml/pipelines/components/estimators/regressors/lightgbm_regressor.py

```diff
@@ -70,10 +70,14 @@ def __init__(self, boosting_type="gbdt", learning_rate=0.1, n_estimators=20, max
 
     def _encode_categories(self, X, fit=False):
         """Encodes each categorical feature using ordinal encoding."""
-        X_encoded = _convert_to_woodwork_structure(X)
-        X_encoded = _rename_column_names_to_numeric(X_encoded)
-        cat_cols = list(X_encoded.select('category').columns)
-        X_encoded = _convert_woodwork_types_wrapper(X_encoded.to_dataframe())
+        X = _convert_to_woodwork_structure(X)
```

Comment:
Blargh. I'd like to combine lightgbm classifier and regressor, since they share a lot of the same core logic. Probs better to file than to address here tho 😬